components/combobox/src/auro-combobox.js

```diff
@@ -558,14 +549,14 @@ export class AuroCombobox extends LitElement {
     // handle the menu event for an option selection
     this.menu.addEventListener('auroMenu-selectedOption', () => {
       if (this.menu.optionSelected) {
-        const [selected] = this.menu.optionSelected;
+        const selected = this.menu.optionSelected;
 
-        if (!this.optionSelected || this.optionSelected[0] !== selected) {
-          this.optionSelected = [selected];
+        if (!this.optionSelected || this.optionSelected !== selected) {
+          this.optionSelected = selected;
         }
 
-        if (!this.value || this.value[0] !== this.optionSelected[0].value) {
-          this.value = [this.optionSelected[0].value];
+        if (!this.value || this.value !== this.optionSelected.value) {
+          this.value = this.optionSelected.value;
```

Comment:
Also does this break multi-select?

Reply:
No, combobox never supported multiselect